src/gui/owncloudgui.cpp

```diff
@@ -40,11 +40,99 @@
 #include <QScreen>
 
 #ifdef WITH_LIBCLOUDPROVIDERS
+#include "graphapi/drives.h"
 #include "libcloudproviders/libcloudproviders.h"
 #endif
 
 using namespace std::chrono_literals;
 
+namespace {
+
+using namespace OCC;
+
+void addFolder(AccountStatePtr accountStatePtr, const QString &localFolder, const QString &remotePath, const QUrl &webDavUrl)
+{
+    // TODO: workaround: the folder is not auto-created... is this intentional behavior?
+    QDir().mkdir(localFolder);
```

Comment:
The folder is not automatically created as we want to be able to detect a removed folder.

Reply:
So you're saying this line of code should be there? I couldn't find how the old wizard solved this in a reasonable amount of time, and just hacked some `mkdir` into a location that seemed sensible.